website/content/en/docs/deployment/terraform-s3-backend.md

```diff
@@ -6,14 +6,20 @@ weight = 70
 
 ## Local vs. Remote state
 
-While Terraform manages the state of resources it has created through a `terraform.tfstate`, by default this file only exists locally.
-This means that you will need to manually copy over the original `terraform.tfstate` file when managing previously created resources on a different host. 
-This can become difficult to manage and keep in sync when multiple copies exist between different hosts and users.
+While Terraform manages the state of resources it has created through a `terraform.tfstate` file, by default this file only exists locally.
+This means that you will need to manually copy over the original `terraform.tfstate` file when managing previously created resources on a different host. Without having a local copy of the `terraform.tfstate` file, Terraform will attempt to recreate all the existing resource since it does not know their current state.
 
-By using a remote backend, such as AWS S3, state is consolidated in one shared remote location and can be re-used between multiple hosts. Additionally, the state will not be lost if the local `terraform.tfstate` file was accidentally deleted.
+Having multiple copies of the same `terraform.tfstate` can become difficult to manage and keep in sync between different hosts and users.
+Instead, by using a remote backend, such as AWS S3, state is consolidated in one shared remote location and can be re-used between multiple hosts. Additionally, the state will not be lost if the local `terraform.tfstate` file was accidentally deleted.
 
 For additional details on using AWS S3 as a Terraform backend, refer to the following Terraform [documentation](https://developer.hashicorp.com/terraform/language/settings/backends/s3#s3).
 
+> **Important**
+>
+> Required Terraform variables will still need to be provided as input when deploying even if a remote state file is being used.
+>
+> For example, a cluster name will still need to be provided through the TF_VAR_cluster_name or `.tfvar` file before deploying.
```

Comment:
suggest adding another line.

Make sure you provide your `.tfvars` file in the deployment folder and export all the `TF_VAR` variables when making changes to an existing deployment.

Reply:
done